README.md

```diff
@@ -45,6 +47,13 @@ Online signing happens in two situations:
 
 In practice online signing happens at least every three days because of online signature expiry.
 
+#### Publishing and automated testing
+
+Online signing leads to a "preproduction" staging deployment at https://sigstore.github.io/root-signing-staging/.
+This is a fully functional TUF reppository that is then used to run both generic TUF client tests and
+Sigstore specific client tests (with cosign and other sigstore clients). Successful tests lead to a
+"production" staging deployment at https://tuf-repo-cdn.sigstage.dev/.
```

Comment:
```suggestion
"pre-production" staging deployment at https://tuf-repo-cdn.sigstage.dev/.
```
or perhaps "production-like" ?

Reply:
Yeah, I'm trying to find good terms for the test deployment (in this case github.io) and the actual deployment (in this case sigstage.dev)... preferably terms we could use in both staging and production. Who knew naming is hard

Reply:
"pre-prod" &  "final"?

"testing" & "final"?

Reply:
I went with `"testing" deployment` and `"final" deployment` -- not great but arguably better than using the word "production" in there

Reply:
I think that works. "Testing staging" and "final staging" / "testing production" and "final production" seem clear and concise. Nice.